sites/default/sqlconf.php

```diff
@@ -21,3 +21,5 @@
 
 //$config for default site files should never be changed.
 $config = 0; 
+
```

Comment:
No. It is not to be even part of your commit........ you should consider placing it in gitignore for further contributions.